docs/testrunner-toolkit.md

```diff
@@ -146,6 +146,7 @@ Browser support for each framework is based on the Sauce Labs docker images prov
 
 |Playwright Version|Supported Browsers|
 |-----|----|
+|1.11.1|Please see [release notes](https://github.com/saucelabs/sauce-playwright-runner/releases/tag/v1.12.0)|
```

Comment:
Another double-check on this....first column is 1.11.1, but the link tag is 1.12.0. Is that correct?

Reply:
correct. It's `1.11.1` for playwright framework version, and `1.12.0` for playwright runner version